### PR TITLE
3.17.0 Changelog

### PR DESCRIPTION
**Pull request by rileykarson**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,26 @@
 ## 3.17.0 (Unreleased)
+
+FEATURES:
+* **New Resource:** `google_bigquery_dataset_access` ([#1924](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1924))
+* **New Resource:** `google_dialogflow_intent` ([#1936](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1936))
+* **New Resource:** `google_os_login_ssh_public_key` ([#1922](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1922))
+
+IMPROVEMENTS:
+* `compute`: Added update support for `google_compute_interconnect_attachment` `admin_enabled` ([#1931](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1931))
```
Comment (megan07) [inline CHANGELOG.md:9]:
```suggestion
* compute: Added update support for `google_compute_interconnect_attachment` `admin_enabled` ([#1931](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1931))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
+* accesscontextmanager: added `spec` and `use_explicit_dry_run_spec` to `google_access_context_manager_service_perimeter` to test perimeter configurations in dry-run mode. ([#1940](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1940))
+* compute: Added field `log_config` to `google_compute_health_check` and `google_compute_region_health_check` to enable health check logging. ([#1934](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1934))
+* compute: Added more import formats for `google_compute_instance` ([#1933](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1933))
+* sourcerepo: allowed `google_sourcerepo_repo` `pubsub_configs.topic` to accept short topic names in addition to full references. ([#1938](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1938))
+
+BUG FIXES:
+* `compute`: Fixed diff on default value for `google_compute_interconnect_attachment` `admin_enabled` ([#1931](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1931))
```
Comment (megan07) [inline CHANGELOG.md:16]:
```suggestion
* compute: Fixed diff on default value for `google_compute_interconnect_attachment` `admin_enabled` ([#1931](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1931))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,26 @@
...
+* `compute`: Fixed perma-diff on `google_compute_interconnect_attachment` `candidate_subnets` ([#1931](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1931))
```
Comment (megan07) [inline CHANGELOG.md:17]:
```suggestion
* compute: Fixed perma-diff on `google_compute_interconnect_attachment` `candidate_subnets` ([#1931](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1931))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,26 @@
 ## 3.17.0 (Unreleased)
+
+FEATURES:
+* **New Resource:** `google_bigquery_dataset_access` ([#1924](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1924))
+* **New Resource:** `google_dialogflow_intent` ([#1936](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1936))
+* **New Resource:** `google_os_login_ssh_public_key` ([#1922](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1922))
+
+IMPROVEMENTS:
+* compute: Added update support for `google_compute_interconnect_attachment` `admin_enabled` ([#1931](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1931))
+* accesscontextmanager: added `spec` and `use_explicit_dry_run_spec` to `google_access_context_manager_service_perimeter` to test perimeter configurations in dry-run mode. ([#1940](https://github.com/terraform-providers/terraform-provider-google-beta/pull/1940))
```
Comment (danawillow) [inline CHANGELOG.md:10]: This should go before the previous one so that the `compute` blocks are grouped

**Pull request merged**